### DISCUSSION TITLE
求问，这个问题的解决方案：Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, actions/setup-node@v3, actions/cache/restore@v3, actions/upload-artifact@v3.

### DISCUSSION THREAD

**Discussion by 2644635282** [tw93/Pake#692] [General]
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, actions/setup-node@v3, actions/cache/restore@v3, actions/upload-artifact@v3.
![Uploading 微信截图_20240327152729.png…]()

**Comment by 2644635282** [comment]
![20240327152729](https://github.com/tw93/Pake/assets/62086344/9be6aef9-3c28-4064-bf27-6ff88d70f11e)

**Comment by tw93** [comment]
看了下你的打包记录，后面都成功了，关于 Node.js 16 actions  这个错误可以不关注，没事的，最近我更新一下版本就好了

**Reply by 1024Boy** [reply]
我也遇到这个问题
